base/int.jl

```diff
@@ -2,23 +2,25 @@
 
 ## integer arithmetic ##
 
-const IntTypes = (Int8, UInt8, Int16, UInt16, Int32, UInt32,
-                  Int64, UInt64, Int128, UInt128)
+const Signed64Types    = (Int8,Int16,Int32,Int64)
```

Comment:
The *64* types are only introduced because the Int128 types currently require a work-around on 32-bit systems (on LLVM 3.3?). I would add a comment in this respect near here, since this is the first place this is relevant.